### PR TITLE
chore: add feeback app contest article

### PR DESCRIPTION
**Pull request by SiR-PENt**
## What does this PR do?

Add feedback app contest article to the blog

### CODE REVIEW

File: apps/formbricks-com/components/shared/AuthorBox.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
-import AuthorJohannes from "@/images/blog/johannes-co-founder-formbricks-small.jpg";
 import Image from "next/image";
 
 interface AuthorBoxProps {
   name: string;
   title: string;
   date: string;
   duration: string;
+  src: string;
```
Comment (jobenjada) [inline apps/formbricks-com/components/shared/AuthorBox.tsx:8]: pls change variable name to something more telling e.g. imageSrc or photoSrc

File: apps/formbricks-com/images/blog/ola-content-writer.jpg
Comment (jobenjada) [inline apps/formbricks-com/images/blog/ola-content-writer.jpg:1]: legend!

File: apps/formbricks-com/pages/blog/best-feedback-app-and-how-to-use-them/index.mdx [hunk 2]
```diff
@@ -0,0 +1,351 @@
+import Image from "next/image";
+import LayoutMdx from "@/components/shared/LayoutMdx";
+import AuthorBox from "@/components/shared/AuthorBox";
+import AuthorOla from "@/images/blog/ola-content-writer.jpg";
+import Formbricks from "./formbricks-open-source-feedback-app.png";
+import Userpilot from "./userpilot-feedback-app.png";
+import Pendo from "./free-survey-tool-limesurvey-open-source-software-opensource.png";
```
Comment (jobenjada) [inline apps/formbricks-com/pages/blog/best-feedback-app-and-how-to-use-them/index.mdx:7]: "Pendo" but image name says its limesurvey

File: apps/formbricks-com/pages/blog/best-feedback-app-and-how-to-use-them/index.mdx [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+import Appcues from "./appcues-feedback-app.png";
+import Survicate from "./survicate-feedback-app.png";
+import Qualaroo from "./qualaroo-feedback-app.png";
+
+export const meta = {
+  title: "Feedback App Contest: 7 Candidates, 1 Winner (and how to use it)",
+  description:
+    "Only when you understand your users and customers, they will come back and tell others about your service. ",
```
Comment (jobenjada) [inline apps/formbricks-com/pages/blog/best-feedback-app-and-how-to-use-them/index.mdx:15]: Please add more keywords here e.g. "We looked at the best in app feedback tools 2024 and found a clear winner. Gather feedback in your app for free with Formbricks."

**Review by jobenjada — CHANGES_REQUESTED** [review]
Overall it looks great, will merge after fixed + new header image

**Comment by jobenjada** [comment]
Hey @Olaleye-Blessing thanks for following up.

Two more things:

1. Please resolve this issue (I think its due to your changes in the writer avatar code)
image

2. Please add a few more keywords to the names of these image files:
image

you can see how I did it here:
image

these keywords shouldnt be the same in each file, just write 4-6 into each file name related to the articles topic.

Thanks!!

**Comment by Olaleye-Blessing** [comment]
Hello @jobenjada, you tagged the wrong person :-). I'm sure you wanted to tag @SiR-PENt :).

**Comment by SiR-PENt** [comment]
@jobenjada, concerning the first issue, I think we have to find another way other than passing the `src` as prop. This will keep throwing this error as long as we make `imageSrc` a string in the `AuthorBoxProps` interface. 

I'm still trying to find my way around it though

**Comment by SiR-PENt** [comment]
@jobenjada, I have added a temporary solution to this that will work just fine for both of us. Although, it might get cumbersome if there are more writers.

**Review by jobenjada — APPROVED** [review]

**Pull request merged**